docker/jupyterlab/azure-pipelines.yml

```diff
@@ -213,6 +221,14 @@ jobs:
             ${{ variables.imageTagLatest }}
             ${{ variables.imageTagBeforeVulnerabilityScan }}
 
+      # Trivy vulnerability scan, includes python packages installed in image
+      - task: CmdLine@2
+        displayName: "Trivy vulnerability scan final image"
+        inputs:
+          script: |
+            trivy image --exit-code 0 --severity LOW,MEDIUM ${{ variables.imageName }}:${{ variables.imageTagLatest }}
+            trivy image --exit-code 1 --severity HIGH,CRITICAL ${{ variables.imageName }}:${{ variables.imageTagLatest }}
```

Comment:
Vil foreslå at du legger til opsjonene `--light --ignore-unfixed`
--light gir mer kompakt output, og --ignore-unfixed gjør at bygget ikke feiler på sikkerhetshull som det ikke er publisert fiks for ennå. Det er kun de tilfellene hvor de finnes en fiks vi kan gjøre noe med det.